jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -611,6 +611,11 @@ public Builder withScopeManager(ScopeManager scopeManager) {
       return this;
     }
 
+    public Builder withMDcScopeManager() {
```

Comment:
* tracer already has the ability to accept any other type of ScopeManager, there's no need to explicitly introduce methods for MDC to it
  * most people install tracer via the Configuration object so the ability to switch to MDC-enabled scope manager should be there. We could start with just a method to add a custom scope manager that people can call programmatically (config-based enabling of MDC manager can be done later if requested)

Reply:
Removed